keyboards/abacus/rules.mk

```
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
```

```suggestion
BOOTLOADER = atmel-dfu
```
Sorry. As @noroadsleft pointed out, I was wrong about Elite-C, and atmel-dfu was correct. I apologize for the trouble.